distribution/docs/src/main/resources/_contents/_catalog-frameworks/global-attribute-validators-definition-contents.adoc

```diff
@@ -40,6 +40,8 @@ Each object in the list of validators is the validator name and list of argument
 [WARNING]
 ====
 The value of the `arguments` key must always be an array of strings, even for numeric arguments, e.g. `["1", "10"]`
+The value of the `validators` must always be an array of validators.
```

Comment:
It might be a bit confusing to put this here because readers could think this applies to the root `validators` key (also, the `validators` key for `match_any` hasn't been introduced yet). This might not even be necessary since I think your explanation of `validators` in `match_any` is sufficient.